src/main/java/homework_3/ImmutableClass.java

```diff
@@ -0,0 +1,43 @@
+package homework_3;
+
+/*
+
+The class is immutable because:
+
+The instance variable of the class is final i.e. we cannot change the value of it after creating an object.
+The class is final so we cannot create the subclass.
+There is no setter methods i.e. we have no option to change the value of the instance variable.
+
+These points makes this class as immutable.
+
+
+ */
+
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+
+public final class ImmutableClass {
```

Comment:
must be method which return modified (new) object yours class

Reply:
Thanks a lot!

Reply:
Good job!